gcc/rust/ast/rust-ast.cc

```diff
@@ -4223,26 +4223,32 @@ Attribute::is_parsed_to_meta_item () const
 }
 
 void
-BlockExpr::strip_tail_expr ()
+BlockExpr::try_convert_last_stmt ()
 {
-  if (expr)
+  rust_assert (expr == nullptr);
+  // HACK: try to turn the last statement into a tail expression
+  if (statements.size () && statements.back ()->is_expr ())
```

Comment:
I think doing
```
if (!statements.empty() && statements.back ()->is_expr ()) {
}
```

I prefer using empty instead of a size check is more readable

Reply:
Fixed in [26fe2f9](https://github.com/Rust-GCC/gccrs/pull/2274/commits/26fe2f9120795cb8b53a53c39f82904570cde8d9).